go.mod

```diff
@@ -784,3 +788,5 @@ replace github.com/docker/distribution v2.8.3+incompatible => github.com/docker/
 
 // ARO-RP sub-packages
 replace github.com/Azure/ARO-RP/pkg/api => ./pkg/api
+
+replace kubevirt.io/containerized-data-importer-api => kubevirt.io/containerized-data-importer-api v1.55.0
```

Comment:
How often will we need to update this API version?

Reply:
we need to fix the version because some incompatibilities with the k8s client on the original version

```
kubevirt.io/containerized-data-importer-api@v1.63.1/pkg/apis/core/v1beta1/types.go:82:19: undefined: corev1.VolumeResourceRequirements

kubevirt.io/containerized-data-importer-api@v1.63.1/pkg/apis/core/v1beta1/types.go:105:24: undefined: corev1.TypedObjectReference

kubevirt.io/containerized-data-importer-api@v1.63.1/pkg/apis/core/v1beta1/zz_generated.deepcopy.go:1837:17: undefined: v1.TypedObjectReference

FAIL    github.com/Azure/ARO-RP/pkg/monitor/cluster [build failed]
```